src_c/event.c

```diff
@@ -144,6 +144,19 @@ RemovePending_PGS_VIDEORESIZE_Events(void * userdata, SDL_Event *event)
     return 1;
 }
 
+static int SDLCALL
+RemovePending_PGS_VIDEOEXPOSE_Events(void * userdata, SDL_Event *event)
```

Comment:
This should be probably be called: `_pg_remove_pending_PGS_VIDEOEXPOSE`.

Because of the naming conventions:
- `_` prefix because it's for private internal use.
- `pg` prefix because it's a pygame function.

Reply:
Yeah, but I had just copy pasted fro my the functions name above, let’s me make that modification to that function too

Reply:
Did it :)